variables.tf

```diff
@@ -38,6 +38,11 @@ variable "storage_type" {
   default     = "standard"
 }
 
+variable "storage_encrypted" {
+  description = "(Optional) Specifies whether the DB instance is encrypted. The default is false if not specified."  
+  default     = false
```

Comment:
Use strings "false" and "true" for boolean values, _e.g._ `default     = "false"`.
Terraform does not work with plain booleans correctly in many cases, and they recommend using strings.